utils.py

```diff
@@ -88,7 +88,7 @@ def times():
     return int(round(time.time()))
 
 
-DEBUG_MODE = False
+DEBUG_MODE = not os.getenv('NO_DEBUG_MODE')
```

Comment:
Fixed it for now, but maybe this is not the cleanest way?

Reply:
Doesn't this always set it to `True` ?

Reply:
I don't think so, it's what we are already doing: https://github.com/hedyorg/hedy/blob/6e7573d8710d90079385614e617f0581d7bde732/app.py#L2908-L2912 . Just that this get executed before

Reply:
I think the better solution is probably to control the initialization order of all these things better, by making the tables class members of Database and initializing it when the app is created

Reply:
Yeah, that makes sense too.